gst/nnstreamer/tensor_common.c

```diff
@@ -40,13 +41,13 @@ static const gchar *tensor_element_typename[] = {
   [_NNS_FLOAT32] = "float32",
   [_NNS_INT64] = "int64",
   [_NNS_UINT64] = "uint64",
-  [_NNS_END] = NULL,
```

Comment:
let's add null [_NNS_MAX] = NULL

Reply:
Right, It will be updated. Actually, I'm not sure whether `_NNS_MAX` is really necessary.